jib-gradle-plugin/README.md

```diff
@@ -220,12 +221,12 @@ Property | Type | Default | Description
 `volumes` | `List<String>` | *None* | Specifies a list of mount points on the container.
 `workingDirectory` | `String` | *None* | The working directory in the container.
 
-<a name="extradirectory-closure"></a>`extraDirectory` is an object with the following properties (see [Adding Arbitrary Files to the Image](#adding-arbitrary-files-to-the-image)):
+<a name="extradirectories-closure"></a>`extraDirectories` is an object with the following properties (see [Adding Arbitrary Files to the Image](#adding-arbitrary-files-to-the-image)):
 
-Property | Type
---- | ---
-`path` | `File`
-`permissions` | `Map<String, String>`
+Property | Type | Default | Description
+--- | --- | --- | ---
+`paths` | `Object` | `(project-dir)/src/main/jib` | Extra directories acceptable by [`Project.files()`](https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html#files-java.lang.Object...-), such as `String`, `File`, `Path`, `List<String\|File\|Path>`, etc. Can be absolute or relative to the project root.
+`permissions` | `Map<String, String>` | *None* | Permission map for the files added from extra directories.
```

Comment:
We should explicitly say that it maps from file path on container to unix permissions, and maybe even say that it defaults to 644 for files and 755 for folders.

Reply:
Updated. Please see if this sounds good.

Reply:
Sounds good to me :+1: